src/vivarium/framework/configuration.py

```diff
@@ -63,6 +68,6 @@ def _get_default_specification():
 
     user_config_path = Path('~/vivarium.yaml').expanduser()
     if user_config_path.exists():
-        model_specification.configuration.update(user_config_path, layer='component_configs')
+        model_specification.configuration.update(user_config_path, layer='base')
```

Comment:
We may need to think about this more closely later, but setting at the component_configs level didn't make sense.  From a users perspective, every simulation they run has these values set.  We perhaps need a fifth `user_defaults` level.